### PR TITLE
feat(release): update helm charts for v0.8.35

### PR DESCRIPTION
**Pull request by RyanHolstien**

### CODE REVIEW

**Comment by RyanHolstien** [comment]
Verified install on test instance

**Review by dexter-mh-lee — APPROVED** [review]

File: charts/datahub/quickstart-values-with-neo4j.yaml [hunk 1]
```diff
@@ -102,4 +102,4 @@ global:
 
     managed_ingestion:
       enabled: true
-      defaultCliVersion: "0.8.34.1"
+      defaultCliVersion: "0.8.35.1"
```
Comment (dexter-mh-lee) [inline charts/datahub/quickstart-values-with-neo4j.yaml:105]: This cli version should be 0.8.35 ?

File: charts/datahub/values.yaml [hunk 2]
```diff
@@ -156,7 +156,7 @@ global:
 
     managed_ingestion:
       enabled: true
-      defaultCliVersion: "0.8.34.1"
+      defaultCliVersion: "0.8.35.1"
```
Comment (dexter-mh-lee) [inline charts/datahub/values.yaml:159]: same here

File: charts/datahub/README.md [hunk 3]
```diff
@@ -101,6 +101,6 @@ helm install datahub datahub/datahub --values <<path-to-values-file>>
 | global.datahub.managed_ingestion.enabled | bool | `true` | Whether or not UI-based ingestion experience is enabled. |
 | global.datahub.encryptionKey.secretRef | string | `datahub-encryption-secrets` | The reference to a secret containing an alpha-numeric encryption key, which is used to encrypt Secrets on DataHub. If a secret reference is not provided, a random one will be generated for you in a Kubernetes secret named `datahub-encryption-secrets`. |
 | global.datahub.encryptionKey.secretKey | string | `encryption_key_secret` | The key of a secret containing an alpha-numeric encryption key, which is used to encrypt Secrets on DataHub. If a secret reference is not provided, a random one will be generated for you in a Kubernetes secret value named `encryption_key_secret` within a secret named `datahub-encryption-secrets`. |
-| global.datahub.managed_ingestion.defaultCliVersion | string | `0.8.34.1` | This is the version of the DataHub CLI to use for UI ingestion, by default. |
+| global.datahub.managed_ingestion.defaultCliVersion | string | `0.8.35.1` | This is the version of the DataHub CLI to use for UI ingestion, by default. |
```
Comment (dexter-mh-lee) [inline charts/datahub/README.md:104]: nit: need to update this

**Review by dexter-mh-lee — APPROVED** [review]

**Pull request merged**